### PR TITLE
Code changes for SNAP-2989: Snappy UI rebranding to Tibco ComputeDB iff it's Enterprise Edition 

### PR DESCRIPTION
**Pull request by snappy-sachin**
## Changes proposed in this pull request

1. Setting product name to SnappyData if it is community edition else TIBCO ComputeDB.
2. Loading GFXD version properties to identify correct product type

## Patch testing

 - Tested Manually

## ReleaseNotes.txt changes

 Yes

## Other PRs 

 **Spark:** https://github.com/SnappyDataInc/spark/pull/151

(Does this change require changes in other projects- store, spark, spark-jobserver, aqp? Add the links of PR of the other subprojects that are related to this change)

### CODE REVIEW

File: cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala [hunk 1]
```diff
@@ -159,13 +163,26 @@ class LeadImpl extends ServerImpl with Lead
...
+        "io/snappydata/SnappyDataVersion.properties")
+      GemFireVersion.createVersionFile()
+      GemFireVersion.getInstance(classOf[GemFireXDVersion], SharedUtils.GFXD_VERSION_PROPERTIES)
+      if (GemFireVersion.isEnterpriseEdition) {
+        "TIBCO ComputeDB"
+      } else {
+        "SnappyData"
+      }
+    }
+
```
Comment (dshirish) [inline cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala]: We can use `SnappySession.isEnterpriseEdition` here.

Comment (snappy-sachin) [reply cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala]: Done

**Comment by snappy-sachin** [comment]
> LGTM. Added one comment. Please get the screenshots reviewed from all relevant folks.

Done.

**Comment by snappy-sachin** [comment]
All requested changes and feedback are taken care. Hence merging.

**Pull request merged**